### ISSUE TITLE
Suggest alternative tasks that use Open Source softwares

### ISSUE THREAD

**Issue by dannycolin** [OpenScienceMOOC/Main#18]
As proposed in #17 It would be a good idea to propose at least one Open Source alternative (with a tasks/guide) for every non-free softwares we are suggesting in a module.

For example, we have a task in Module 5 on how to use GitHub. The goal will be to also create a similar tasks with a open source softwares like GitLab (or an other FOSS alternative).

**Comment by Protohedgehog** [comment]
So this is a great idea, I think. In the main text for Module 5 we do mention this, but just not explicitly in the tasks. Yet. Do you think we should do the same sort of screencast efforts for open alternatives too? I should note that for some, like Publons, there simply aren't any open alternatives too. I think being more explicit about these things is a great idea though.

**Comment by dannycolin** [comment]
> Do you think we should do the same sort of screencast efforts for open alternatives too?

You mean the audio version? To give an example, what I had in mind is to do somthing like https://github.com/OpenScienceMOOC/Module-5-Open-Research-Software-and-Open-Source/blob/master/content_development/Task_1.pdf but with an open alternative.

> I should note that for some, like Publons, there simply aren't any open alternatives too.

Of course, if there isn't a open alternative (yet) I think it's totally ok to only have a task with a non-free software.

**Comment by Protohedgehog** [comment]
OK, gotcha! The thing is, I'm not too familiar with GitLab, so this would either take a lot of learning on my part, or getting someone already familiar with it to do the text/audio/screencast guide. Do you think simply mentioning it in the existing tasks could be sufficient? That would be much more simple.

**Comment by dannycolin** [comment]
I could probably do the text part during the holiday or we could also send a message on slack (maybe twitter) to see if someone would be willing to help with that issue.

**Comment by Protohedgehog** [comment]
Good idea! Lemme ask around too and see what we can conjour up. Perhaps we could also just pilfer this? https://www.youtube.com/watch?v=enMumwvLAug

**Comment by dannycolin** [comment]
Looks good to me. Don't know if it's up-to-date tho they've redesign the website a bit. But yea, that's a start. ;)
